src/lib/crypto/test/AESTests.cpp

```diff
@@ -328,6 +328,7 @@ void AESTests::testCBC()
 	}
 }
 
+#ifndef WITH_BOTAN
```

Comment:
I believe this will give an undefined reference to this function while linking.
Maybe its better to just remove the registration of the testcase in the header file.

```
#ifndef WITH_BOTAN
        CPPUNIT_TEST(testECB);
#endif
```

Reply:
Same for the `testECB()` testcase in:
src/lib/crypto/test/DESTests.h

Reply:
I followed your suggestion and compilation/linking is ok now.
The tests fail on the src/lib/test though but I am pretty sure that it was the case before my changes.

Reply:
Great! Looks good.